tests/excel/test_excel_integration.py

```
@@ -0,0 +1,335 @@
+        if has_header:
+            df = pd.DataFrame(data[1:], columns=data[0])
+        else:
+            df = pd.DataFrame(data)
+
+        df.to_excel(file_path, index=False, header=has_header)
+        return file_path
+
+    def create_mock_env(self):
+        """Sphinx環境のモックを作成。"""
+
+        class MockConfig:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)